GoogleDataTransport/GDTCORLibrary/Public/GDTCORPlatform.h

```diff
@@ -69,20 +69,52 @@ typedef NS_ENUM(NSInteger, GDTCORNetworkMobileSubtype) {
   GDTCORNetworkMobileSubtypeLTE = 11,
 };
 
+/** The different possible reachabilityFlags option. */
+typedef NS_ENUM(NSInteger, GDTCORReachabilityFlagsOption) {
```

Comment:
This should be `NS_OPTIONS(uint32_t`, and maybe these only need to be declared with the options needed to indicate a wifi connection on watchOS. On non-watchOS, you can probably just typedef.

Reply:
Fixed